underwater_inspection/ViewpointInfo[] vp_info